data/scripts.js

```diff
@@ -652,16 +652,17 @@ let BattleScripts = {
 			}
 			nullDamage = false;
 
-			for (let i = 0; i < damage.length; i++) {
+			for (let j = 0; j < damage.length; j++) {
```

Comment:
Is there a reason this isn't a for...of?

Reply:
Because at the time I originally wrote this loop I didn't know about the `.entries` trick.